src/test/java/io/github/anthorx/parquet/sql/read/RecordConsumerInitializerTest.java

```diff
@@ -0,0 +1,52 @@
+package io.github.anthorx.parquet.sql.read;
+
+import junit.framework.Assert;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.Collections;
+
+@ExtendWith(MockitoExtension.class)
+public class RecordConsumerInitializerTest {
+
+  @Mock
+  private Connection connection;
```

Comment:
No "when" defined on this mock, this mean you pass a null preparedStatement to the PreparedStatementRecordConsumer constructor row 44 of RecordConsumerInitializer. This could result in a test failure later if PreparedStatementRecordConsumer test not null preparedStatement.